pages/dkp/kommander/2.0/workspaces/workspace-platform-services/application-deployment/index.md

```diff
@@ -10,9 +10,10 @@ excerpt: Deploy applications to attached clusters using the CLI
 <!-- markdownlint-disable MD004 MD040 -->
 
 This topic describes how to use the CLI to deploy an application to attached clusters within a workspace.
-To use the Kommander UI to deploy applications, see [Customize a workspace's platform applications](../#customize-a-workspaces-platform-services).
+To use the Kommander UI to deploy applications, see [Customize a workspace's platform applications](../#customize-a-workspaces-platform-application).
 
-See [Workspace Platform Services](../#workspace-platform-applications) for a list of all applications and those that are enabled by default.
+<<<<<<< HEAD
```

Comment:
Seems like a missing resolved rebase.

Reply:
https://github.com/mesosphere/kommander/pull/840

Reply:
OMG, that's actually in the Kommander repo. I will fix it. I guess we'll have to close this PR and run the sync again?!